controls/SV-204619.rb

```diff
@@ -32,7 +32,10 @@
 
   if package('postfix').installed?
     options = { assignment_regex: /^\s*([^=]*?)\s*=\s*(.*?)\s*$/ }
-    pf_config = parse_config_file('/etc/postfix/main.cf', options).params['smtpd_client_restrictions'].split(',')
+
+    if (defined? parse_config_file('/etc/postfix/main.cf', options).params['smtpd_client_restrictions'])then
```

Comment:
`parse_config_file('/etc/postfix/main.cf', options).params['smtpd_client_restrictions']` may not exist. Use conditional statement to confirm its existence and avoid subsequent error in calling method `split()`.